velociraptor/catalogue/registration.py

```diff
@@ -1021,49 +1021,142 @@ def registration_gas_hydrogen_species_masses(
     else:
         raise RegistrationDoesNotMatchError
 
+def registration_cold_dense_gas_properties(
+    field_path: str, unit_system: VelociraptorUnits
+) -> (unyt.Unit, str, str):
+    """
+    Registers the mass of cold (T < 10^4.5 K), dense (nH > 0.1 cm^-3) gas in apertures.
+    """
+    unit = unit_system.mass
 
-def registration_gas_element_ratios_times_masses(
+    # Capture aperture size
+    match_string = (
+        "Aperture_ColdDense([a-zA-Z]*)Masses_aperture_total_gas_([0-9]*)_kpc"
+    )
+    regex = cached_regex(match_string)
+    match = regex.match(field_path)
+
+    if match:
+        quantity_key = match.group(1)
+        aperture_size = match.group(2)
+
+        try:
+            long_quantity = {
+                "DiffuseMetal": "Diffuse Metal",
+                "Gas": "Gas",
+            }[quantity_key]
+            short_quantity = {
+                "DiffuseMetal": "diffuse_metal",
+                "Gas": "gas",
+            }[quantity_key]
+        except KeyError:
+            raise RegistrationDoesNotMatchError
+        
+        full_name = f"""{long_quantity} Masses in Cold, Dense ($T < 10^{{4.5}}\;{{\rm K}}$,  
+        $n_{{\\rm H}}$ > 0.1 \\; {{\rm cm^{{-3}}}}$) Gas ({aperture_size} kpc)"""
```

Comment:
```suggestion
        full_name = (
            f"{long_quantity} Masses in Cold, Dense ($T < 10^{{4.5}}\;{{\rm K}}$, "
            f"$n_{{\\rm H}}$ > 0.1 \\; {{\rm cm^{{-3}}}}$) Gas ({aperture_size} kpc)"
        )
```

Otherwise you'll have a `\n` in your caption.